### PR TITLE
Day 4 Excercise

### PR DESCRIPTION
**Pull request by dodoph**

### CODE REVIEW

File: Day4_excercise.rb [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+symbols = ["@", "%", "*", "!"]
+uppercharacters = ("A".."Z").to_a
+lowercharacters = ("a".."z").to_a
+hasNumber = false
+hasSymbol = false
+hasUpper = false
+hasLower = false
+hasMoreThan8Characters = false
+password.each_char do |element| #switch string to chars
+  numbers.each do |number|
```
Comment (xsuoownit) [inline Day4_excercise.rb:13]: These checks all look similar except the array you use is different. Maybe you can make it a method.

File: practice.rb [hunk 2]
```diff
@@ -0,0 +1,58 @@
+# print "Please create your password: "
```
Comment (xsuoownit) [inline practice.rb:1]: What is this file for?

Comment (dodoph) [reply practice.rb:1]: what's file?

Comment (xsuoownit) [reply practice.rb:1]: This practice.rb

File: Day4_excercise.rb [hunk 3]
```diff
@@ -0,0 +1,68 @@
+print "Please create your password: "
+password = gets.chomp
+numbers = ["0", "1", "2", "3", "4", "5", "6", "7", "8", "9"]
```
Comment (xsuoownit) [inline Day4_excercise.rb:3]: Does `("0".."9").to_a` works here?

**Pull request merged**